### DISCUSSION TITLE
应用密钥的疑问

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#248] [02.提问]
<img width="1677" alt="image" src="https://user-images.githubusercontent.com/6092525/226512884-0df34a5d-9a9c-48c8-9de7-424a05aa4315.png">
这代地址和名称写死应该不对吧，这样会导致新创建的租户没发登录系统， 如果用这个脚手架做别的系统 域名 应该做成可配置化的更好吧？

**Comment by zuihou** [comment]
这里特指的是 lamp-web和lamp-web-plus 这2个前端，若你二次开发时，修改了这2个值，只需要保证前端里面的配置一致即可。

![image](https://user-images.githubusercontent.com/8500410/226522561-c1810b2c-cdad-41d9-a6c3-a5b2596d70a4.png)

**Reply by zqb666** [reply]
我的意思是 我们在新建租户的时候，该租户也是用的 新的前端工程，那这里的 值写死就不对了 要根据对应租户的clientid和 密钥去生成了，而不是写死在后端代码里

**Reply by zuihou** [reply]
好，下个版本优化下
